modules/learn/pages/data/transactions.adoc

```diff
@@ -45,44 +30,78 @@ If two transactions simultaneously attempt to write to the same document, one is
 Use of transactions requires _Network Time Protocol_ (NTP) to be used to synchronize time across all cluster-nodes.
 See xref:install:synchronize-clocks-using-ntp.adoc[Clock Sync with NTP], for details.
 
-[#indexes-and-transactions]
 === Services and Transactions
 
 The indexes provided by the Index, Search, and Analytics Services are _not_ atomically updated with the commits performed by transactions: however, they are updated with https://en.wikipedia.org/wiki/Eventual_consistency[Eventual Consistency].
 
 Neither the Query Service nor the Search Service sees uncommitted data.
 However, https://jepsen.io/consistency/models/snapshot-isolation[Snapshot Isolation] is _not_ provided: consequently, if a transaction performs a commit while a query or search is ongoing, then the query or search may return data from both prior to and subsequent to the commit.
 
-Note that the Query Service provides the xref:n1ql:n1ql-rest-api/index.adoc#table_xmr_grl_lt[At_Plus] feature, which allows queries to wait for indexes to be appropriately updated, following a transaction.
 
-[#limitations]
+// == Custom Metadata
+include::java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[tag=custom-metadata]
+
+
 == Limitations
 
-Transactions, which constitute an evolving technology, currently have the following limitations:
+Couchbase Distributed ACID Transactions currently have the following limitations:
 
 * Only documents whose initial size is 10 MB or less can be included in a transaction.
 
 * Non-transactional updates should not be made to any document involved in a transaction while the transaction is itself in progress: this prevents the non-transactional update from being overwritten.
 
-* The number of writes required by a transactional update is greater than the number required for a non-transactional update: a transactional update requires writes in order to stage and commit data, and also to maintain transaction records.
-Consequently, transactional updates may be less performant than non-transactional updates.
+* The number of writes required by a transactional update is greater than the number required for a non-transactional update.
```

Comment:
Hmm, I'm not sure this is a limitation.. Maybe we need a separate section..

Reply:
(I'm very aware of the optics of the current very large limitations section.)

Reply:
Shuffled sections